hair_salon/__manifest__.py

```diff
@@ -17,6 +17,8 @@
         'sales_team',
         'website_appointment',
         'theme_orchid',
+        "base_automation",
```

Comment:
we try to follow alphabetical order (except for theme and studio)

Reply:
Really? 
```bash
>>> str('base_automation') > str('base_geolocalize')
False
```

Reply:
Oupsi